README.md

```diff
@@ -1,128 +1,104 @@
-# IBM Cloud <module_name> - Terraform Module
```

Comment:
This comment is related to test.yml file under .github/workflows

Please add code snippet to write test resuts into slack channel


copy the code from https://github.com/terraform-ibm-modules/terraform-ibm-satellite/blob/main/.github/workflows/test.yml

From line #45 to #56

Reply:
Added the 45 to 46 lines